src/main/scala/net/snowflake/spark/snowflake/Utils.scala

```diff
@@ -160,75 +160,6 @@ object Utils {
   def makeTempPath(tempRoot: String): String =
     Utils.joinUrls(tempRoot, UUID.randomUUID().toString)
 
```

Comment:
Explain to reviewers:
These two functions are not used. Firstly, I want to keep them since they are in Utils. But these two functions look like legacy functions for external stage. I think we'd better to remove them.